lib/vlocity.js

```diff
@@ -125,8 +125,12 @@ Vlocity.runDataPacksCommand = async function(action, options) {
 
             process.chdir(options.chdir);
         }
+        var passedInOptions = JSON.parse(JSON.stringify(options)),
+            vlocity_run = new Vlocity(passedInOptions);
```

Comment:
Definitely this new Vlocity should not be moved here.

Reply:
All the ones in this file are for VBT and can be ignored anyway.

Reply:
Changes have been made in vlocityCli.js to track actual command provided by user.  Now UI changes to send config variable(disableVbtAnalytics) can be ignored.